builtin-func.l

```diff
@@ -386,9 +387,11 @@ int main(int argc, char* argv[])
 
 			fprintf(fp_zeek_init, "# %s\n\n", auto_gen_comment);
 			fprintf(fp_func_def, "// %s\n\n", auto_gen_comment);
+			fprintf(fp_func_h, "#pragma once\n\n");
 			fprintf(fp_func_h, "// %s\n\n", auto_gen_comment);
```

Comment:
Would flip auto_gen_comment and `pragma once`, so that the note about auto generation stays at the first line.

Reply:
Here and elsewhere.